apps/framework-cli/src/framework/controller.rs

```diff
@@ -198,11 +198,6 @@ pub(crate) async fn create_or_replace_tables(
             )
         })?;
 
-    // Clickhouse doesn't support dropping a view if it doesn't exist, so we need to drop it first in case the schema has changed
-    if !PROJECT.lock().unwrap().is_production {
-        drop_tables(fo, configured_client).await?;
-    }
```

Comment:
Hum you do need to drop the tables if the data models have changed in between restarts

Reply:
That didn't seem to be the case for me.  See my earlier comment.
https://github.com/514-labs/moose/pull/711#issuecomment-2021143183